HyperPlatform/util.cpp

```diff
@@ -49,16 +50,30 @@ _Must_inspect_result_ _IRQL_requires_max_(DISPATCH_LEVEL) NTKERNELAPI
 using MmAllocateContiguousNodeMemoryType =
     decltype(MmAllocateContiguousNodeMemory);
 
-// dt nt!_LDR_DATA_TABLE_ENTRY
-struct LdrDataTableEntry {
+// from Windows Research Kernel v1.2
+// stable fields for Win 7+
+struct KLdrDataTableEntry {
```

Comment:
Why this is changed? Not only those changes are unused, those are not even correct. I would hope to see this commit is rolled back.

Reply:
Of course, I will roll this back.   

Please let me explain why I proposed the change.

Code in "_**UtilpInitializeRtlPcToFileHeader**_" references the _DriverSection_ via the _PDRIVER_OBJECT_ from the "**_DriverEntry_**" parameter.  

The _DriverSection_ is a struct KLDR_DATA_TABLE_ENTRY (not a LDR_DATA_TABLE_ENTRY).  The 
[Windows Internal 7](https://books.google.com/books?id=y83LDgAAQBAJ&pg=PT287&lpg=PT287&dq=PsLoadedModuleList+KLDR_DATA_TABLE_ENTRY&source=bl&ots=kHcrStUjow&sig=ACfU3U3dmLK_TUaXkyAVht717ju9QwMO9Q&hl=en&sa=X&ved=2ahUKEwiEj7HF3avjAhUJH80KHfJVDb4Q6AEwCHoECAgQAQ#v=onepage&q=PsLoadedModuleList%20KLDR_DATA_TABLE_ENTRY&f=false) book sample describes this.

I agree that my PR struct KLDR_DATA_TABLE_ENTRY is not perfect.  Every source I found has a slightly different representation of this struct.  see [DarthTon/Blackbone](https://github.com/DarthTon/Blackbone/blob/master/src/BlackBoneDrv/NativeStructs.h)
and [MiLoadImageSection](a1eded2d8efb071685e1f3cc59a1054f8545b73a)

Code in "**_UtilpUnsafePcToFileHeader_**" references DllBase and SizeOfImage from KLDR_DATA_TABLE_ENTRY.  I wanted to help to make these offsets correct.

Reply:
Thanks for explanation and trying to correct the offsets. I missed that the structure was (is) incorrect from the first place. 

In this case, it is more than welcome to accept this change. I would only ask to rename `__Unused5` to `unused`.

Reply:
Did more research - found improved KLDR_DATA_TABLE_ENTRY.  Fixed __Unused5